package.json

```diff
@@ -20,6 +20,7 @@
     "allotment": "^1.19.3",
     "axios": "^1.6.0",
     "date-fns": "^2.30.0",
+    "deep-object-diff": "^1.1.9",
```

Comment:
Do we really need this new dependency to implement this functionality?

Reply:
yes. I am using diff() function to compare the initial state with the current state to disable or unable the Update button: 
<img width="712" alt="Screenshot 2024-08-14 at 22 38 35" src="https://github.com/user-attachments/assets/f7992ebd-1bf9-4184-bcac-cf941a0cfd4f">

Reply:
removed diff() and created the following function:
![photo_2024-08-16_14-56-57](https://github.com/user-attachments/assets/28de6c8f-f3df-4075-bf59-aa016c617277)